src/mo_frontend/printers.ml

```diff
@@ -189,7 +189,6 @@ let string_of_symbol = function
   | X (N N_seplist_dec_field_semicolon_) -> "seplist(<dec_field>,<semicolon>)"
   | X (N N_seplist_exp_ob__COMMA_) -> "seplist(<exp(ob)>,,)"
   | X (N N_seplist_exp_field_semicolon_) -> "seplist(<exp_field>,<semicolon>)"
-  | X (N N_separated_nonempty_list_semicolon_exp_field_) -> "seplist+(<exp_field>,<semicolon>)"
```

Comment:
I think this will need a corresponding new clause. It only gets triggered when the parser wants to emit a syntax hint.

Reply:
I didn't understand this comment. Is it resolved?

Reply:
Nope, you can crash the interpreter with a syntax error, when it tries to report it.

Reply:
I'll try to fix now.

Reply:
Looks like there is already a clause (you have reused one), so no problem:
``` Motoko
> { a and a with true };
stdin:1.16-1.20: syntax error [M0001], unexpected token 'true', expected one of token or <phrase> sequence:
  seplist1(<exp_field>,<semicolon>) }
```

Reply:
You added it in this PR, actually. Cool, resolved!